.gitignore

```diff
@@ -24,3 +24,5 @@ htmlcov/
 .vscode/
 *.swp
 *.swo
+
+.env
```

Comment:
🔒 Good practice! Adding '.env' to .gitignore prevents sensitive environment variables from being committed.